### PR TITLE
Add SatoshiSwap volume

### PR DESCRIPTION
**Pull request by satoshiswapcore**
https://satoshiswap.exchange

Deployed on CoreDAO mainnet https://defillama.com/chain/CORE

### CODE REVIEW

**Comment by llamatester** [comment]
The satoshiswap adapter exports: 
        
 
 _______________________________________
    Dexs for 12/4/2023
    _______________________________________
    
    Failed to get total volume on core: Type `Query` has no field `uniswapFactories`: {"response":{"errors":[{"locations":[{"line":2,"column":3}],"message":"Type `Query` has no field `uniswapFactories`"}],"status":200,"headers":{}},"request":{"query":"query get_total_volume($block: Int) { \n  uniswapFactories(\n    block: { number: $block }\n    ) {\n      totalVolumeUSD\n    }\n     }","variables":{"block":2688715}}}
    Failed to get daily volume on core: Type `Query` has no field `uniswapDayData`: {"response":{"errors":[{"locations":[{"line":2,"column":5}],"message":"Type `Query` has no field `uniswapDayData`"}],"status":200,"headers":{}},"request":{"query":"query get_daily_volume($id: Int) { \n    uniswapDayData (id: $id) {\n          dailyVolumeUSD\n      } }","variables":{"id":19459}}}
    Failed to get daily volume via alternative query on core: Type `Query` has no field `uniswapDayDatas`: {"response":{"errors":[{"locations":[{"column":7,"line":2}],"message":"Type `Query` has no field `uniswapDayDatas`"}],"status":200,"headers":{}},"request":{"query":"{\n      uniswapDayDatas(where: {date: 1681257600}) {\n          date\n          dailyVolumeUSD\n      }\n  }"}}
    CORE 👇
    Backfill start time: 5/4/2023
    NO METHODOLOGY SPECIFIED
    Timestamp: 1681343998
    Block: 2688715
    Total volume: undefined
    Daily volume: undefined
    
    
    
    
    Done in 4.80s.

File: dexs/satoshiswap/index.ts [hunk 1]
```diff
@@ -0,0 +1,13 @@
+// https://subgraph.satoshiswap.exchange/subgraphs/name/pancakeswap/exchange
+import { univ2Adapter } from "../../helpers/getUniSubgraphVolume";
+import { CHAIN } from "../../helpers/chains";
+
+const endpoints = {
+  [CHAIN.CORE]: "https://subgraph.satoshiswap.exchange/subgraphs/name/pancakeswap/exchange",
+};
+
+const adapter = univ2Adapter(endpoints, {});
```
Comment (dtmkeng) [inline dexs/satoshiswap/index.ts]: could you fix by 
```
const adapter = univ2Adapter(endpoints, {
  factoriesName: "pancakeFactories",
  dayData: "pancakeDayData",
});
```

**Review by dtmkeng — CHANGES_REQUESTED** [review]

**Comment by dtmkeng** [comment]
could you check sync supraph data there is volume just `1680825600`

**Comment by satoshiswapcore** [comment]
Done, waiting for https://github.com/DefiLlama/DefiLlama-Adapters/pull/5919 to be merged as well

File: dexs/satoshiswap/index.ts (hunk 1 above)
Comment (ghost) [reply dexs/satoshiswap/index.ts]: Done

**Comment by llamatester** [comment]
The satoshiswap adapter exports: 
        
 
 _______________________________________
    Dexs for 12/4/2023
    _______________________________________
    
    CORE 👇
    Backfill start time: 7/4/2023
    NO METHODOLOGY SPECIFIED
    Timestamp: 1681343998
    Block: 2688715
    Total volume: 0.3192514231772406
    Daily volume: 0
    
    
    
    
    Done in 4.79s.

**Comment by dtmkeng** [comment]
subgraph function `pancakeDayData` have only one record on  1680825600 could you check?

**Comment by satoshiswapcore** [comment]
@dtmkeng Now we have two records, since we are a new project not many people using the exchange and I am adding it in advance.

**Pull request merged**